src/polyswarm_api/__init__.py

```diff
@@ -923,27 +936,23 @@ def scan_directory(self, directory, recursive=False):
         """
         return self.loop.run_until_complete(self.ps_api.scan_directory(directory, recursive))
 
-    def search_hashes(self, hashes, hash_type='sha256'):
+    def search_hashes(self, hashes):
         """
         Scan a collection of hashes using the PS API synchronously.
 
         :param hashes: Hashes to scan.
-        :param hash_type: Hash type [sha256|sha1|md5]
-        :param rescan: Whether to initiate a rescan for fresh results
         :return: JSON report file
         """
-        return self.loop.run_until_complete(self.ps_api.search_hashes(hashes, hash_type))
+        return self.loop.run_until_complete(self.ps_api.search_hashes(hashes))
 
-    def search_hash(self, to_scan, hash_type='sha256'):
+    def search_hash(self, to_scan):
```

Comment:
we want to force the user to specify the type of hash. And if they do not specify the type of hash then we will assume sha256. So, I need you to re-add the "hash_type" arg to all of these functions. Then update the docs to tell the user to specify the hash type on the command line. And if no hash type is provided, then we will assume sha256.